ui/raidboss/data/02-arr/raid/t4.txt

```diff
@@ -9,7 +9,7 @@
 123.0 "Dreadnaught (NW)" sync / 03:........:Added new combatant Clockwork Dreadnaught\./ window 130,0
 123.0 "4x Bug (S)"
 
-183.0 "2x Rook (E/W center)" sync / 03:........:Added new combatant Spinner-rook\./ window 190,0
+183.0 "2x Rook (E/W)(center)" sync / 03:........:Added new combatant Spinner-rook\./ window 190,0
```

Comment:
I think the original line was intended.  There's two rooks, and they appear east and west of the center, but not very far east or west.  I think it'd be fine to just make this `(E/W)` or `(center)`.  I think `(E/W)(center)` is less understandable, though.

Reply:
Sure, maybe just `(center)` is ok.

Reply:
I can't remember the spawn locations, but I think that if they are close enough to be picked up with a normal tank AoE, it should be labeled as `center`. If not, give the directions so the tank can anticipate their locations.

Reply:
Yeah, I think these are far enough apart that one tank can't pick both of them up.  I'll change this to be E/W and then merge this.  Thanks for the feedback.